license.txt

```diff
@@ -1,4 +1,4 @@
-Copyright 2022 Universal Cannabis Labeling API
+Copyright 2022 Universal Cannabis Labeling API. Some portions copyright 2021 Lucid Green, Inc.
```

Comment:
This relates to a comment I added below, but since you'll probably read this first I was wondering what exactly a Lucid Green copyright means here - does this mean you own the rights to the data models, or just that you've contributed them? Does this prevent another vendor from adopting the standard and returning your data structures?